test/SConstruct

```diff
@@ -18,29 +18,29 @@ if env["target"] in ["editor", "template_debug"]:
     doc_data = env.GodotCPPDocData("src/gen/doc_data.gen.cpp", source=Glob("doc_classes/*.xml"))
     sources.append(doc_data)
 
-if env["platform"] == "macos":
-    library = env.SharedLibrary(
-        "project/bin/libgdexample.{}.{}.framework/libgdexample.{}.{}".format(
-            env["platform"], env["target"], env["platform"], env["target"]
-        ),
+if env["platform"] == "macos" or env["platform"] == "ios":
+    # Static libraries (.dylib) are not supported on the iOS app store.
```

Comment:
```suggestion
    # Dynamic libraries (.dylib) are not supported on the iOS app store.
```

Static libraries (.a) are supported on iOS, as well as dynamic `.frameworks` and `.xcframeworks` with both.

Reply:
This would be good to document somewhere better than a single comment.